clonelists/Nintendo - Wii U (Redump).json

```diff
@@ -171,6 +185,13 @@
 				{"searchTerm": "Mario & Sonic at the Sochi Olympic"}
 			]
 		},
+		{
+			"group": "Mario vs. Donkey Kong - Tipping Stars",
```

Comment:
`Mario vs. Donkey Kong - Tipping Stars` isn't in the Redump DAT, so should be omitted.

Reply:
Same as above.
My bad.

Reply:
I've fixed this.